packages/curve-ui-kit/src/widgets/Footer/Sections.tsx

```diff
@@ -75,7 +75,8 @@ export const getSections = (): SectionProps[] => [
       },
       {
         label: t`Integrations`,
-        href: '#/integrations',
+        // Would've loved to have used ROUTE.PAGE_INTEGRATIONS, but they differ per app.
```

Comment:
Can we somehow make them the same per app?
Currently, this is losing the selected network and switching to eth (at least for DEX).
It might be necessary to pass the `integrationUrl` or `networkName` to the `getSections` function

[Screencast from 18-12-24 09:21:29.webm](https://github.com/user-attachments/assets/8fd65578-956a-4a4f-aa0f-ead1658eb81d)

Reply:
I couldn't really get the `selectedApp` from the header as it's set there and couldn't pass it to the footer easily, so for now I've settled with drilling down `networkName`. If present in the link together with a non http link, it'll use that `networkName` in the generated url. It's almost like `externalAppUrl` from `PageTabs.tsx`, but without the app. Getting tired of this issue lol.